modules/insights-operator.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="insights-operator_{context}"]
+= Insights Operator
+
+[discrete]
+== Purpose
+
+The Insights Operator gathers {product-title} configuration data and sends it to Red Hat. The data is used to produce insights about potential issues that a cluster might be exposed to. These insights are communicated to cluster administrators through Insights Advisor on link:https://console.redhat.com/[console.redhat.com].
```

Comment:
"The data is used to produce proactive insights recommendations about potential issues"

Add reference to Remote health monitoring. Insights operator is complementary to telemeter.

Reply:
@radekvokal Added a "Notes" section referencing Telemetry and the link. I also updated the description with the proactive part.

Reply:
Thanks! 
/lgtm